### DISCUSSION TITLE
Child_process can't execute command with LATIN1 encondig

### DISCUSSION THREAD

**Discussion by FeehGb** [nodejs/help#5005] [Q&A]
### Version

v14.17.4

### Platform

Linux desenvolvimentointerno 5.4.0-96-generic #109-Ubuntu SMP Wed Jan 12 16:49:16 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

### What steps will reproduce the bug?

```javascript

const { exec } = require("child_process");
const string = 'ééé'

const buffer = require('buffer');
const latin1Buffer = buffer.transcode(Buffer.from(string), "utf8", "latin1");
const latin1String = latin1Buffer.toString();

exec('echo '+latin1String+'-ééé  > output6.txt', {encoding:'latin1'})

```

### How often does it reproduce? Is there a required condition?

Always

### What is the expected behavior?

The expected behavior

``` 
ééé-Ã©Ã©Ã©
```

### What do you see instead?

The output terminal econding in LATIN1 
``` 
ï¿½ï¿½ï¿½Ã©Ã©Ã©
``` 
The output terminal in UTF-8
```
���-ééé
```

**Comment by bnoordhuis** [comment]
```js
const latin1Buffer = buffer.transcode(Buffer.from(string), "utf8", "latin1");
const latin1String = latin1Buffer.toString();
```
This doesn't do what I think you think it does. toString() expects UTF-8 but there's no UTF-8 in latin1Buffer contains and that's why it returns those three Unicode "REPLACEMENT CHARACTER" (U+FFFD) characters.

I'll convert this to a discussion.

**Reply by FeehGb** [reply]
There is a way to run NODE with latin1 like PHP ?

**Comment by FeehGb** [comment]
I found a way to do it;
Just save a file in ISO-8859-1, in node use latin1 and exec the file;
I dont know about performance but it works;
